test/tests/functional/pbs_que_resc_usage.py

```diff
@@ -90,12 +91,11 @@ def test_resc_assigned_set_unset(self):
         self.assertEqual(
             int(q_status[0]['resources_available.ncpus']), 8, self.err_msg)
         # job submission
+        t1 = time()
         j_attr = {'Resource_List.ncpus': '3'}
         j1 = Job(attrs=j_attr)
-        j1.set_sleep_time(30)
         jid_1 = self.server.submit(j1)
         j2 = Job(attrs=j_attr)
-        j2.set_sleep_time(30)
```

Comment:
Since changing sleep time doesn't assure that this issue will not come again in slower machines, I thought it is better to keep the default time. By the time we reach the point where we are checking for jobs to get finished (Line number 125), the sleep time remaining is 30 seconds only.
If waiting for 30seconds seems long, is it better to delete the job before checking for jobs to be finished, that is after [this](https://github.com/openpbs/openpbs/pull/2333/files#diff-d0647820564bc73a1d77a2a409b2317af14fe29f8df034ab4d312c70102959a5L121-L122?) line we can do a qdel of the jobs?
What do you think?

Reply:
It depends on why the test is waiting for the job to finish.  Does it need to accumulate resources, or does it just care the job is done?  If it just cares the job is done, then deleting it is fine.

Reply:
The tests in the end just need the job to be completed. It is not testing for accumulation of resources.